### DISCUSSION TITLE
Adding padding for images passed to the Avatar component

### DISCUSSION THREAD

**Discussion by mschasz** [twilio-labs/paste#3500] [General] [answered]
Hi,

Per [this thread](https://twilio.slack.com/archives/CFS9UGBSB/p1695172675792999), I'd like to see if it's possible to add some padding to the image passed to an `Avatar` component. As we migrate to Paste, the images in the Avatar look more zoomed in than before, making the Avatars for sources and destinations look a bit off. The avatars on the left are in Evergreen and those on the right are in Paste.

Thanks!

<img width="544" alt="image" src="https://github.com/twilio-labs/paste/assets/109534751/c439236d-0772-4a1d-9ca2-416f152af41e">

**Comment by TheSisb** [comment]
Hi @mschasz,

Interesting question, this hasn't come up before. I notice in your example screenshot, the images on the left are different from the ones on the right. Do you have a true comparison image using the same avatar images?

There is currently no way to provide `padding` to Avatar for images at this time. Avatar images are sized to fit flush to the Avatar container, usually because the provided images are either too big or too small, and need to be sized correctly. On the other hand, icons and text are centered within the Avatar, as [can be seen on the docs](https://paste.twilio.design/components/avatar).

An option if the images are not user generated is you could edit the image so it fits the way you would like.

**Reply by mschasz** [reply]
Gotcha, here are the same avatar images below (you can see some weird cropping going on).
Evergreen:
<img width="62" alt="image" src="https://github.com/twilio-labs/paste/assets/109534751/33da57ca-bba3-42aa-8462-e12f97d3cf1d">

Paste:
<img width="65" alt="image" src="https://github.com/twilio-labs/paste/assets/109534751/74fd90ab-8aee-494e-93c9-5425cd7dcff2">

**Reply by TheSisb** [reply]
Hi @mschasz,

Just following up that we're still looking into this. Will report back when we conclude on a direction. Thanks in advance for your patience!

**Reply by serifluous** [reply, marked as answer]
Hey @mschasz ! @ceyerkes is going to work on standardizing the images used for sources and destinations in Segment, so you can sync up with her on moving forward with this.

**Reply by lnamba** [reply]
Hi @serifluous and all, wanted to check in on what the outcome of this was? I have a design which requires padding for `Avatar`s as well and I'm not seeing a way to add it. For reference, here is the design:
<img width="193" alt="Screenshot 2024-11-18 at 12 04 13 PM" src="https://github.com/user-attachments/assets/4a31c91d-f605-4e68-9b24-30a44e8b4617">
Here is how it looks with the actual components:
<img width="127" alt="Screenshot 2024-11-18 at 12 05 19 PM" src="https://github.com/user-attachments/assets/bc144487-662e-4cbb-a2a7-b89e9779125d">
cc: @ceyerkes
